notebooks/mobility_model.ipynb

```diff
@@ -0,0 +1,340 @@
+{
```

Comment:
1. generate alpha0 closer to cell where it is first used
2. update params with re-computed alpha1